build.sbt

```diff
@@ -16,9 +16,9 @@ name := "firrtl"
 
 version := "1.2-SNAPSHOT"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.6"
 
-crossScalaVersions := Seq("2.11.12", "2.12.4")
+crossScalaVersions := Seq("2.12.6", "2.11.12")
```

Comment:
Does this order matter?

Reply:
It doesn't matter for `sbt`, but for `mill`, we use the first entry as the default, and since I'm bumping the 2.12 version and changing that line anyway ...

Reply:
Ah I see, thanks for the clarification!